docs/src/pages/index.tsx

```diff
@@ -53,32 +43,36 @@ function Feature({ imageUrl, title, description }) {
     </div>
   );
 }
-
+*/
 function Home() {
   const context = useDocusaurusContext();
   const { siteConfig = {} } = context;
   return (
     <Layout
-      title={`Hello from ${siteConfig.title}`}
+      title={`Grape Design System`}
       description="Description will go into a meta tag in <head />"
     >
-      <header className={clsx("hero hero--primary", styles.heroBanner)}>
+      <header className={clsx("hero", styles.heroBanner)}>
         <div className="container">
           <h1 className="hero__title">{siteConfig.title}</h1>
-          <p className="hero__subtitle">{siteConfig.tagline}</p>
-          <div className={styles.buttons}>
+          <p className="hero__subtitle">{`Design, build and create experiences for Grape.`}</p>
+          <p>{`Aurora was created by Grape for Grape.`}</p>
+          <Link to={useBaseUrl("docs/")}>{`Read the guidelines.`}</Link>
+          {/*<div className={styles.buttons}>
             <Link
               className={clsx(
-                "button button--outline button--secondary button--lg",
+                "button button--secondary button--lg",
                 styles.getStarted
               )}
-              to={useBaseUrl("docs/")}
+              to={useBaseUrl("docs/doc2")}
             >
-              Get Started
+              Read the guidelines
```

Comment:
Do we even need this page? it has almost no content?

Reply:
Yes, it's the homepage. Needs further customization when the font issues are fixed.